media-source-respec.html

```diff
@@ -2169,8 +2252,10 @@ <h2>TextTrack Extensions</h2>
       <h2><dfn data-export="">Byte Stream Formats</dfn></h2>
       <p>The bytes provided through {{SourceBuffer/appendBuffer()}} for a <a>SourceBuffer</a> form a logical byte stream. The format and
         semantics of these byte streams are defined in <dfn id="byte-stream-format-specs">byte stream format specifications</dfn>.
-        The byte stream format registry [[MSE-REGISTRY]] provides mappings between a MIME type that may be passed to {{MediaSource/addSourceBuffer()}} or
-        {{MediaSource/isTypeSupported()}} and the byte stream format expected by a <a>SourceBuffer</a> created with that MIME type. Implementations are encouraged to register
+        <!-- BIG TODO: verify the following paragraph text is good -->
```

Comment:
oops - I accidentally left this note to self in the merge commit. I'll update this PR to remove it.